whereby/sdk
# Listen for client_kicked events and update state accordingly

browser-sdk: Adds `kicked` to `ConnectionStatus`, and removes the duplicate definition from `browser-sdk`
core: listens for `client_kicked` signal events and updates the room state accordingly

### To test
- console log out the room status in the sample-app under the `useRoomConnection` hook - https://github.com/whereby/sdk/blob/kevinhanna%2Fcob-449-listen-for-client_kicked-in-the-sdk/apps/sample-app/src/App.tsx#L60

- join a room
- join the same room from PWA as host
- kick the SDK client
you should see the status update to `kicked`

![image](https://github.com/whereby/sdk/assets/138459303/a4d7142b-f730-4bcf-8c70-3d67f3e8072d)
